v4/internal/otel_expect.go

```diff
@@ -0,0 +1,40 @@
+package internal
+
+import (
+	"go.opentelemetry.io/otel/api/trace/testtrace"
+)
+
+// OpenTelemetryExpect implements internal.Expect for use in testing.
+type OpenTelemetryExpect struct {
+	*testtrace.Tracer
+}
+
+// ExpectTxnEvents TODO
+func (e *OpenTelemetryExpect) ExpectTxnEvents(t Validator, want []WantEvent) {}
+
+// ExpectSpanEvents TODO
+func (e *OpenTelemetryExpect) ExpectSpanEvents(t Validator, want []WantEvent) {}
+
+// ExpectCustomEvents TODO
+func (e *OpenTelemetryExpect) ExpectCustomEvents(t Validator, want []WantEvent) {}
+
+// ExpectErrors TODO
+func (e *OpenTelemetryExpect) ExpectErrors(t Validator, want []WantError) {}
+
+// ExpectErrorEvents TODO
+func (e *OpenTelemetryExpect) ExpectErrorEvents(t Validator, want []WantEvent) {}
+
+// ExpectMetrics TODO
+func (e *OpenTelemetryExpect) ExpectMetrics(t Validator, want []WantMetric) {}
+
+// ExpectMetricsPresent TODO
+func (e *OpenTelemetryExpect) ExpectMetricsPresent(t Validator, want []WantMetric) {}
+
+// ExpectTxnMetrics TODO
+func (e *OpenTelemetryExpect) ExpectTxnMetrics(t Validator, want WantTxn) {}
+
+// ExpectTxnTraces TODO
+func (e *OpenTelemetryExpect) ExpectTxnTraces(t Validator, want []WantTxnTrace) {}
+
+// ExpectSlowQueries TODO
+func (e *OpenTelemetryExpect) ExpectSlowQueries(t Validator, want []WantSlowQuery) {}
```

Comment:
Thanks for providing this. Just to make sure I understood correctly: I assume from these we will only implement `ExpectSpanEvents`, `ExpectMetrics`, `ExpectMetricsPresent` and `ExpectTxnMetrics`?

Reply:
I'm glad you brought that up, cuz I was thinking of implementing different ones than that! I had written up comments that included that information, but it seems the disappeared when I later made some changes. I'll re-add them.